src/OrchardCore.Modules/OrchardCore.Localization/Permissions.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using OrchardCore.Security.Permissions;
+
+namespace OrchardCore.Localization
+{
+    public class Permissions : IPermissionProvider
+    {
+        public static readonly Permission ManageCultures = new Permission("ManageCultures", "Manage supported culture");
```

Comment:
Manage supported cultures

Reply:
Well here it should state in the description that it is for managing default and supported cultures.